docker/entrypoint.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+gcloud auth activate-service-account --key-file=/secrets/gcloud_key.json
+#sleep 3600
+export GOOGLE_APPLICATION_CREDENTIALS=/secrets/gcloud_key.json
+export BUCKET=$1
+export UBMLEXP_GIT=$2
+export WSN=$3
+export FILELIST_TRAIN=$4
+export FILELIST_VAL=$5
+export NAME=$6
+export CONFIG=$7
+export BUCKET_LOCAL=$8
+export RESULT_DIR=${BUCKET_LOCAL}/results
+export DATA_DIR=${BUCKET_LOCAL}/data
+export MODEL_DIR=${BUCKET_LOCAL}/models
+export LOG_DIR=${RESULT_DIR}/${NAME}/logs
+export CHECKPOINT_PATH=${RESULT_DIR}/${NAME}/checkpoints
+export WARM_START_NAME=${MODEL_DIR}/${WSN}
+#./uberduck-ml-exp/experiments/taco2_lj_lachow/config.json
+git clone $UBMLEXP_GIT
+#sleep 3600
+gcsfuse --implicit-dirs $BUCKET $BUCKET_LOCAL
+#sleep 3600
```

Comment:
remove these comments for cleanliness? or are those uncommented lines sometimes needed? if they are sometimes needed, can you write a `# NOTE(sam):` and explain why?